exercises/tweet.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import redis
+import sys
+
+if len(sys.argv) != 3 and len(sys.argv) != 4:
+    print("wrong number of parameters. got {} arguments expecting 2 or 3".format(len(sys.argv) - 1))
+    exit(1)
+
+user = sys.argv[1]
+command = sys.argv[2]
+# print(f"user:{user} is executing {command} command")
+if len(sys.argv) == 4:
+    content = sys.argv[3]
+    # print(f"content is {content}")
+
+try:
+    conn = redis.Redis()
+    conn.ping()
+except:
+    print("tweeter server is down, please try again later")
+    exit(1)
+
+if command == "post":
+    if conn.exists(f"{user}_followers") != 1:
```

Comment:
The Redis convention uses a colon, not an underscore, between parts of the key...